compiler-core/src/format/tests/use_.rs

```diff
@@ -214,7 +214,7 @@ fn arity_1_access_call() {
         r#"pub fn main() {
   use x, y <- promise.await(
     file.read()
-    |> promise.map(something),
+      |> promise.map(something),
```

Comment:
Oh dear, I've been shortsighted. I'm not a fan of how it looks with use.

Perhaps your previous suggestion of only doing it when there's multiple arguments is better?

Reply:
Ok! I've made it so that pipes/binary operators won't get nested if they're the only argument of a function call or the only item of a list/tuple